models/meteora/meteora_trades.sql

```diff
@@ -0,0 +1,159 @@
+ {{
+  config(
+        schema = 'meteora',
+        alias = 'trades',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['tx_id', 'outer_instruction_index', 'inner_instruction_index', 'tx_index','block_month'],
+        pre_hook='{{ enforce_join_distribution("PARTITIONED") }}',
```

Comment:
lets remove this until we're given reason to add it (aka it won't run without it)

Reply:
remove all four of these lines?

Reply:
no, i meant just line 11, not sure why the comment shows those above it too

Reply:
we need lines 8-10